docs/dm/zh_CN/maintenance/dm-ansible.md

```diff
@@ -105,17 +105,27 @@ The key's randomart image is:
 
 以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录。
 
-```
-$ wget http://download.pingcap.org/dm-ansible-latest.tar.gz
-```
+- 下载 1.0.0-alpha 版本
+
+    ```
+    $ wget https://download.pingcap.org/dm-ansible-v1.0.0-alpha.tar.gz
```

Comment:
in *dm-ansible-v1.0.0-alpha.tar.gz*, we still have `dm_version = latest`.
so, should we need to update this `dm_version` in *dm-ansible-v1.0.0-alpha.tar.gz* and re-release it?

Reply:
ping @liubo0127

Reply:
we should do it ASAP